source/Framework Basics.rst

```diff
@@ -0,0 +1,173 @@
+Framework Basics
+=================
+
+Interventions
+-------------
+Interventions can help you execute specific actions based on the current state of the simulation. The simulation engine of Bharat-Sim checks if an inactive intervention can be activated based on the given conditions during every tick. All the active interventions will be recorded in the context and the user can query the context using ``context.activeInterventionNames.contains(interventionName)``.
```

Comment:
Please use the correct capitalisation of BharatSim (as opposed to Bharat-Sim).

Reply:
Actually, this section is still in progress. I thought that git push means it will push only to my forked repo. I don't remember creating a pull request after I modified the Intervention.rst file. I  will look into all the suggestions and will definitely do a spell check before I create a pull request again.

Reply:
No problems, just make these small changes and I'll merge them. Things are mostly ok!